schema/schema.yaml

```diff
@@ -589,4 +589,86 @@ tables:
         expression: radial_velocity_error_kms >= 0
 
 
-version: "v0.1"
\ No newline at end of file
+  - name: CompanionRelationships
+    "@id": "#CompanionRelationships"
+    description: Relationships between sources
+    primaryKey: 
+    - "#CompanionRelationships.source"
+    - "#CompanionRelationships.companion" # or companion_name
+
+    columns:
+    - name: source
+      "@id": "#CompanionRelationships.source"
+      datatype: string
+      length: 100
+      description: Main identifier for an object; links to Sources table
+      ivoa:ucd: meta.id;meta.main
+      nullable: false
+    - name: companion
+      "@id": "#CompanionRelationships.companion"
+      datatype: string
+      length: 100
+      description: Identifier for a companion object
```

Comment:
Maybe clarify this can be an external identifier? Ie, not one that needs to be in the Sources table.